weaviate/collections/classes/config.py

```diff
@@ -735,6 +746,31 @@ def mistral(
                 The maximum number of tokens to generate. Defaults to `None`, which uses the server-defined default
         """
         return _GenerativeMistral(model=model, temperature=temperature, maxTokens=max_tokens)
+    
+    @staticmethod
+    def nvidia(
+        *,
+        base_url: Optional[str] = None,
+        model: Optional[str] = None,
+        temperature: Optional[float] = None,
+        max_tokens: Optional[int] = None,
+    ) -> _GenerativeProvider:
```

Comment:
weaviate/collections/classes/config_named_vectors.py - could you also add the new module here?

Reply:
And could you please add tests here: test/collection/test_config.py